### PR TITLE
Improved popup text for state "non internet" (EXPOSUREAPP-4569)

### PR DESCRIPTION
**Pull request by janetback**
### Description
Improved popup for state 'non internet' / aligned popup text with iOS.
-> Bug fix for EXPOSUREAPP-4569

device-2021-01-22-131604

### CODE REVIEW

**Review by ralfgehrer — DISMISSED** [review]

**Review by AlexanderAlferov — DISMISSED** [review]

**Comment by harambasicluka** [comment]
Lint error, as the string before had a placeholder which was removed by this change. Someone from us needs to take over.

```
  /home/circleci/project/Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/qrcode/scan/SubmissionQRCodeScanFragment.kt:135: Error: Format string 'submission_error_dialog_web_generic_network_error_body' is not a valid format string so it should not be passed to String.format [StringFormatInvalid]
                  getString(
                  ^
      /home/circleci/project/Corona-Warn-App/src/main/res/values-de/strings.xml:772: This definition does not require arguments
  /home/circleci/project/Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/tan/SubmissionTanFragment.kt:105: Error: Format string 'submission_error_dialog_web_generic_network_error_body' is not a valid format string so it should not be passed to String.format [StringFormatInvalid]
                  getString(
                  ^
      /home/circleci/project/Corona-Warn-App/src/main/res/values-de/strings.xml:772: This definition does not require arguments
  /home/circleci/project/Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/testresult/pending/SubmissionTestResultPendingFragment.kt:140: Error: Format string 'submission_error_dialog_web_generic_network_error_body' is not a valid format string so it should not be passed to String.format [StringFormatInvalid]
          getString(
          ^
      /home/circleci/project/Corona-Warn-App/src/main/res/values-de/strings.xml:772: This definition does not require arguments
```

**Comment by harambasicluka** [comment]
@axelherbstreith please test if the strings still make sense in all locations and provide some screenshots. thanks :)

**Comment by MikeMcC399** [comment]
If the code is going to be refactored please consider also checking for network status **before** starting a QR code scan or requesting TAN input from the user (see https://github.com/corona-warn-app/cwa-app-android/issues/2182).

**Comment by harambasicluka** [comment]
@MikeMcC399 thanks for pointing to this issue, I'll forward it! :)

**Comment by axelherbstreith** [comment]
Tested the updated texts! The changes make sense in every occurrence 👍

**Review by harambasicluka — DISMISSED** [review]

**Review by ralfgehrer — DISMISSED** [review]

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/qrcode/scan/SubmissionQRCodeScanFragment.kt [hunk 1]
```diff
@@ -132,10 +132,7 @@ class SubmissionQRCodeScanFragment : Fragment(R.layout.fragment_submission_qr_co
             is CwaClientError, is CwaServerError -> DialogHelper.DialogInstance(
                 requireActivity(),
                 R.string.submission_error_dialog_web_generic_error_title,
-                getString(
-                    R.string.submission_error_dialog_web_generic_network_error_body,
-                    exception.statusCode
-                ),
+                getString(R.string.submission_error_dialog_web_generic_network_error_body),
```
Comment (ralfgehrer) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/qrcode/scan/SubmissionQRCodeScanFragment.kt:135]: You could actually drop the getString(), can't you?

Comment (axelherbstreith) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/qrcode/scan/SubmissionQRCodeScanFragment.kt]: Good point! Is removed in all files

**Review by harambasicluka — APPROVED** [review]

**Review by ralfgehrer — APPROVED** [review]

**Pull request merged**